<?xml version="1.0" encoding="UTF-8"?>
<!--
    $Author: kevin-hejl  
    $Date: 2014-6-23 下午02:41:46  
    $Revision: 1.0  
    $Purpose: 总账-关联科目 
-->
<bm:model xmlns:e="aurora.service.exception" xmlns:o="aurora.database.local.oracle" xmlns:bm="http://www.aurora-framework.org/schema/bm" xmlns:f="aurora.database.features" alias="t1" baseTable="CUX_ACCOUNTING_DOC_ACCOUNT">
    <bm:fields>
        <bm:field name="accounting_doc_type_id" databaseType="NUMBER" datatype="java.lang.Long" physicalName="ACCOUNTING_DOC_TYPE_ID" prompt="单据类型ID"/>
        <bm:field name="account_id" databaseType="NUMBER" datatype="java.lang.Long" physicalName="ACCOUNT_ID" prompt="科目ID"/>
    </bm:fields>
    <bm:features>
        <f:standard-who/>
    </bm:features>
    <bm:primary-key>
        <bm:pk-field name="accounting_doc_type_id"/>
        <bm:pk-field name="account_id"/>
    </bm:primary-key>
    <e:exception-descriptor-config>
        <e:exception-descriptor exception="java.sql.SQLException" handleClass="aurora.database.SQLExceptionDescriptor">
            <e:error-message code="1" message="科目代码不能重复"/>
        </e:exception-descriptor>
    </e:exception-descriptor-config>
</bm:model>
